dist/README.md

```diff
@@ -1,7 +1,53 @@
-## Running ovn-kubernetes with ovsdb-etcd
-
-In order to run [ovn-kubernetes](https://github.com/ovn-org/ovn-kubernetes) with ovsdb-etcd:
-* build the ovsdb-etc image and push it to a common image repository e.g. docker hub or quay
-* update the image name in the ovn-kubernetes/dist/templates/ovnkube-db.yaml.j2 (will be set by kind.sh arguments)
-* replace OVNk files by corresponded files from the ovn-kubernetes directory.
-* run kind.sh to deploy ovn-kubernetes 
\ No newline at end of file
+# Deplyment of ovsdb-etcd on ovn-kubernetes
+
+1. prepare ovn-kubernetes
+	- clone `ovn-kubernetes` from the main repository:
+		```bash
+		# Clone ovn-kubernetes repo
+		mkdir -p $HOME/work/src/github.com/ovn-org
+		cd $HOME/work/src/github.com/ovn-org
+		git clone https://github.com/ovn-org/ovn-kubernetes
+		cd ovn-kubernetes
+		```
+	- build the ovn-kubernetes container image.
+		```bash
+		pushd go-controller
+		make
+		popd
+		pushd dist/images
+		make fedora
+		popd
+		```
+1. prepare ovsdb-etcd
+	- clone `ovsdb-etcd` from the main repository:
+		```bash
+		# Clone ovsdb-etcd repo
+		git clone https://github.com/IBM/ovsdb-etcd
+		cd ovsdb-etcd
+		```
+	- login to your account of common image repository e.g. docker hub or quay.
+	- update the image name on the following files (replace `omerbh\etcd` and `omerbh\server` with your image names,one for the server and one for the etcd):
+		- `tagDocker`
+		- `pushDocker`
+		- `dist/ovn-kubernetes/dist/templates/ovnkube-db.etcd.yaml.j2`
+		- `dist/ovn-kubernetes/dist/templates/ovnkube-db.yaml.j2`
+		- `dist/deployment/ovnkube-db.yaml`
```

Comment:
why do we need this file

Reply:
this part has been removed from the docs.